docs/pages/machine-id/reference/cli.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Machine ID CLI Reference
+description: CLI reference for Teleport Machine ID.
+---
+
+# Machine ID CLI Reference
+
+## `tbot start`
+
+Starts the Machine ID client `tbot`, fetching and writing certificates to disk
+at a set interval.
+
+<Details scope={["cloud"]} scopeOnly={true}>
+```code
+$ tbot start \
+   --data-dir=/var/lib/teleport/bot \
+   --destination-dir=/opt/machine-id \
+   --token=00000000000000000000000000000000 \
+   --join-method=token \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=example.teleport.sh:443
+```
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+```code
+$ tbot start \
+   --data-dir=/var/lib/teleport/bot \
+   --destination-dir=/opt/machine-id \
+   --token=00000000000000000000000000000000 \
+   --join-method=token \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=auth.example.com:3025
+```
+</Details>
+
+| Flag                 | Description                                                                                    |
+|----------------------|------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                              |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                     |
+| `-a/--auth-server`   | Address of the Teleport Auth Server (On-Prem installs) or Proxy Server (Cloud installs).       |
+| `--token`            | A bot join token, if attempting to onboard a new bot; used on first connect.                   |
+| `--ca-pin`           | CA pin to validate the auth server.                                                            |
+| `--data-dir`         | Directory to store internal bot data. Access should be limited. See `tbot init` for details.   |
```

Comment:
`tbot init` doesn't (currently at least) touch the internal data dir, only destinations, so this might be a bit of a red herring.